flask_smorest/utils.py

```diff
@@ -137,3 +137,38 @@ def prepare_response(response, spec, content_type):
                         field
                     ]
                 ) = response.pop(field)
+
+
+def normalize_config_prefix(config_prefix: str):
+    """Normalize custom API config prefix
+
+    :param str config_prefix: raw prefix
+    :return: normalized prefix. Underscore is automatically added if prefix misses one.
+    """
+    result = config_prefix.strip().upper()
+    if result and not result.endswith("_"):
+        result += "_"
+    return result
+
+
+def get_config_key(*, ctx, key):
+    """Get flask config key potentially affected by config_prefix
+
+    :param flask_smorest.Blueprint|flask_smorest.Api ctx: object
+        containing ``config_prefix`` attribute.
+    :param str key: full flask config key is constructed as ``config_prefix + key``.
+    :return: full flask config key
+    """
+    return getattr(ctx, "config_prefix", "") + key
+
+
+def get_config_value(*, app, ctx, key, default=None):
```

Comment:
Why kw only args?

Reply:
I was using `get_config_value` and I myself forgot which of params is suppose to go first? `app` or `ctx`? So to force myself to use correct parameters I've made them kw only.

It is not a big deal. I can easily change it back.

---

Though, I do not know if this function is going to be useful after I will introduce a `current_api` context variable.

Reply:
Sure. I started with a few inline comments before my longer review message.